package.json

```diff
@@ -9,6 +9,14 @@
   "bin": {
     "linkinator": "build/src/cli.js"
   },
+  "prettier": {
+    "useTabs": false,
```

Comment:
Should be able to trim this to just be:
```
"singleQuote": true,
"trailingComma": "es5",
```

I think the rest are prettier defaults if I remember correctly